doc/quickstart.md

```diff
@@ -14,23 +14,27 @@ See [Setup](./setup.md) for other ways and more details on setting up Lean.
     del elan-init.ps1
     ```
     See the [elan repo](https://github.com/leanprover/elan) for other installation options and details.
-    
+
 1. Install [VS Code](https://code.visualstudio.com/).
 
-1. Open VS Code and install the `lean4` extension.
+1. Create a new folder called `Foo` and type:
+    ```lean
+    lake init Foo
+    ```
 
-    ![installing the vscode-lean4 extension](images/code-ext.png)
+1. Open this folder in VS Code and install the `lean4` extension.
 
-1. Create a new file with the extension `.lean` and add the following code:
-    ```lean
-    import Leanpkg
+    ![installing the vscode-lean4 extension](images/code-ext.png)
 
-    #eval Leanpkg.leanVersionString
-    ```
-    You should get a syntax-highlighted file with a "Lean Infoview" on the right that tells you the installed Lean version when placing your cursor on the last line.
+    You should get a syntax-highlighted file with a "Lean Infoview" on the right.
+    Add the following #eval statement which tells you the installed Lean version
+    when placing your cursor on the last line.
```

Comment:
Why did you remove the code to be typed? Where is the file coming from now, is it opened automatically by VS Code?

Reply:
In the end it's not clear to me if the quickstart should stick with the simpler stand-alone file setup or not.

Reply:
Went with the simpler vscode setup, see new version.